.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo" 
+    directory: "/" 
```

Comment:
Does this need to point to the subcrates we use, or is dependabot smart enough to figure out the root Cargo.toml?

Reply:
Good catch, it is not. I will update the PR to reflect that.

Reply:
Thanks! Took care of it.

On a different note I wasn't aware there were different crates. The README doesn't call out. How about including that within the README? What do you think?

Reply:
Sure, README could be updated with repo structure, sounds like a great idea (for a separate PR).

Reply:
OK, I will do a separate PR. Thanks!